scripts/importer_engine_support_check.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2020 Google LLC
```

Comment:
Because of how go packages work one "directory" becomes one binary, so this should be in a directory.

Also, maybe start this with a verb like

check_importer_matches_engine ?

Reply:
I left it as a single file because I saw this: https://github.com/hashicorp/terraform/blob/master/scripts/generate-plugins.go

I'll move it and rename it.

Reply:
Oh, I wonder if they run it like `go run generate-plugins.go`. In that case it may work even if you have multiple binaries in the same dir...

Reply:
They do run it like that, I think. I ran it like that when it was just one file. I don't mind multiple dirs.